policyengine_us/parameters/gov/states/wv/tax/income/credits/low_income_family_tax_credit/amount/head_of_household.yaml

```diff
@@ -0,0 +1,56 @@
+description: West Virginia allows for this low-income family tax credit amount for head of household filers.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: West Virginia low-income family tax credit for head of household filers
+  reference:
+  - title: Code of West Virginia §11-21-22B (b)
+    href: https://code.wvlegislature.gov/11-21-22B/
+  - title: 2021 West Virginia Personal Income Tax Forms And Instructions-2021 FAMILY TAX CREDIT TABLES
+    href: https://tax.wv.gov/Documents/TaxForms/2021/it140.pdf#page=28 
+
```

Comment:
could you also 2022 the 2022 West Virginia Personal Income Tax Forms And Instructions - for all yaml files - just to make sure that it was not inflation adjusted